sql/core/src/main/scala/org/apache/spark/sql/expressions/Aggregator.scala

```diff
@@ -43,52 +43,52 @@ import org.apache.spark.sql.execution.aggregate.TypedAggregateExpression
  *
  * Based loosely on Aggregator from Algebird: https://github.com/twitter/algebird
  *
- * @tparam I The input type for the aggregation.
- * @tparam B The type of the intermediate value of the reduction.
- * @tparam O The type of the final output result.
+ * @tparam Input The input type for the aggregation.
+ * @tparam Buffer The type of the intermediate value of the reduction.
+ * @tparam Result The type of the final output result.
  * @since 1.6.0
  */
-abstract class Aggregator[-I, B, O] extends Serializable {
+abstract class Aggregator[-Input, Buffer, Result] extends Serializable {
```

Comment:
maybe IN, BUF, and OUT

we have mostly used upper case type parameters in Spark.